.github/actions/create-release-archive/macos/action.yml

```diff
@@ -0,0 +1,38 @@
+name: Create release archive
```

Comment:
Why are we duplicating the complete file for the different systems, can we not use an `if` on the operating system just for the command?

Reply:
I tried but couldn't get it to work. It might of been because I was using 7z incorrectly and the if statements were actually working fine. There is no meaningful error message and iterations take time. I think we should be able reduce duplication now that we have a working version.